docs/source/configuration/settings-reference.md

```diff
@@ -339,6 +339,41 @@ apiExpanders
     ```
     The configuration accepts a list of matchers, with the ability to filter by the request path and action type for maximum flexibility.
     It also accepts a `querystring` object that allows configuring the expanders via query string parameters, such as the navigation expander.
+
+toolbarComponents
```

Comment:
@ksuess is there a typo in here? Should it be `additionalToolbarComponents`?

Reply:
yes indeed, thanks for the hint, @sneridagh !